### PR TITLE
Websockets client support

### PR DESCRIPTION
**Pull request by wtrocki**
Adding support for websockets client with ability to listen for the changes. 
Currently we updating the cache however local component is not receiving the update:
investigating that issue.

### CODE REVIEW

**Comment by danielpassos** [comment]
WoW. Love this!!!!

I see the new user on the console

```
Successfully connected to ws
ListUser.js:42 {allUsers: Array(3)} {subscriptionData: {…}, variables: {…}}
ListUser.js:47 Added new item using subscription {id: "17", name: "User 3", dateOfBirth: "2018-11-01T12:49:10.344Z", version: 1, __typename: "User"}
```

but it does not appear on the screen without I hit the load more button.

**Comment by StephenCoady** [comment]
This is "unreal" :P

playing with it now...

**Comment by wtrocki** [comment]
@danielpassos Yes.. Trying to figure out why this is happening. Cache is getting the data, but component is not rendered again. Following the docs so need to dig deeper.

**Comment by wtrocki** [comment]
I spent some time trying to crack this. I'm doing things exactly like in documentation and couple examples.
Data is actually delivered, but internal Apollo component is not rendering again. 
It looks like apollo issue (found couple issues with refetch method). 
Going to continue fixing this

File: web/src/view/ListUser.js [hunk 1]
```diff
@@ -126,12 +169,11 @@ class UserItem extends React.Component {
     cache.writeQuery({
       query: GET_USERS,
       data: {
-        allUsers: newUsers
+        'allUsers': newUsers
```
Comment (StephenCoady) [inline web/src/view/ListUser.js]: should this be quoted?

**Review by StephenCoady — APPROVED** [review]

**Pull request merged**